corehq/apps/locations/models.py

```diff
@@ -77,6 +77,7 @@ class LocationType(models.Model):
     administrative = models.BooleanField(default=False)
     shares_cases = models.BooleanField(default=False)
     view_descendants = models.BooleanField(default=False)
+    last_modified = models.DateTimeField(auto_now=True)
```

Comment:
my read of https://docs.djangoproject.com/en/1.8/ref/models/fields/#datefield is that this uses the project timezone and not UTC. i was going to wonder if we should change this but i see we use it everywhere in the code so seems ok